index.js

```diff
@@ -57,3 +63,17 @@ function tryParseFloat(value) {
 	}
 	return result;
 }
+
+function getTypeFromUnit ( unit ) {
+	if ( cssAngleUnits.indexOf(unit) !== -1 ) {
+		return 'angle';
+	} else if ( cssResolutionUnits.indexOf(unit) !== -1 ) {
```

Comment:
Your commit got rid of one "else," but I'm suggesting you get rid of ALL else statements. Like so:
```js
if ( cssAngleUnits.indexOf(unit) !== -1 ) {
    return 'angle';
}
if ( cssResolutionUnits.indexOf(unit) !== -1 ) {
    return 'resolution';
}
if ( cssFrequencyUnits.indexOf(unit) !== -1 ) {
    return 'frequency';
}
if ( cssTimeUnits.indexOf(unit) !== -1 ) {
    return 'time';
}
return 'length';
```
In fact, now that I look at this deeper, if you want to have some fun and boost performance, you can create a hash object and do a lookup instead of these "if" statements, like so:
```js
var fill = require('lodash.fill');
var merge = require('lodash.merge');
var zipObject = require('lodash.zipobject');

const lookup = merge(
    createLookups(cssAngleUnits, 'angle'),
    createLookups(cssResolutionUnits, 'resolution'),
    createLookups(cssFrequencyUnits, 'frequency'),
    createLookups(cssTimeUnits, 'time')
);

function createLookups(list, value) {
    return zipObject(list, fill(Array(list.length), value));
}

function getTypeFromUnit(unit) {
    return lookup[unit] || 'length';
}
```
You don't have to if you don't want to. Just sayin'.